api-reference/_hidden/dxSchedulerScrolling/mode.md

```diff
@@ -6,7 +6,15 @@ default: 'standard'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the scrolling mode.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following scrolling modes are available:
+
+- **Standard**       
+All appointments are simultaneously loaded into the DOM, and the whole grid is rendered.
+
+- **Virtual**          
+Appointments are loaded into the DOM when they get into the viewport and removed once they leave it; the grid is rendered within the current viewport only.
+
+[note]The *"virtual"* mode is only available for the *"day"*, *"week"*, and *"workWeek"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/).
```

Comment:
*"Virtual"* mode is only available for *"day"*, *"week"*, and *"workWeek"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/).